denguefever_tw/dengue_linebot/views.py

```diff
@@ -316,3 +316,63 @@ def suggestion_list(request):
 def gov_report_list(request):
     context = {'gov_reports': GovReport.objects.all().order_by('-report_time')}
     return render(request, 'dengue_linebot/gov_report_list.html', context)
+
+
+@login_required
+def push_msg_form(request):
+    context = {'areas': MinArea.objects.all()}
+    return render(request, 'dengue_linebot/push_msg.html', context)
+
+
+@login_required
+def push_msg_result(request):
+    areas_id = request.POST.getlist('msg_to')
+    content = request.POST['content']
+    img = request.POST['img']
+    error_msgs = list()
+    push_logs = list()
+
+    if not areas_id:
+        error_msgs.append('You do not choose any area!')
+    if not content and not img:
```

Comment:
In the condition, neither of them will raise the error.
Maybe i fix it to "You do not write **either** content or image" ?